packages/opentelemetry-exporter-collector/src/transform.ts

```diff
@@ -184,12 +186,45 @@ export function toCollectorSpan(
     droppedAttributesCount: 0,
     events: toCollectorEvents(span.events),
     droppedEventsCount: 0,
-    status: span.status,
+    status: toCollectorStatus(span.status),
     links: toCollectorLinks(span, useHex),
     droppedLinksCount: 0,
   };
 }
 
+/**
+ * Converts StatusCode
+ * @param code
+ */
+export function toCollectorCode(
+  code: StatusCode
+): opentelemetryProto.trace.v1.StatusCode {
+  switch (code) {
+    case 0:
```

Comment:
I think enums will be better instead of hardcoded numeric values.
similar to how it is done [here for spanKind](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-exporter-collector/src/types.ts#L350)